study-builder/studies/rarex/study.conf

```diff
@@ -816,11 +766,12 @@
         "type": "ACTIVITY_INSTANCE_CREATION",
         "activityCode": "PARENTAL_CONSENT"
```

Comment:
Took me a while to wrap my head around the parental thing. Here's the [relevant ticket](https://broadinstitute.atlassian.net/browse/DDP-5696) that I used as reference. Essentially, in English:
```
if self:
  give self-consent

if child_age < 7:
  give consent-parental

if (7 <= child_age < majority) and child_not_capable:
  give consent-parental

if (7 <= child_age < majority) and child_is_capable:
  give consent-assent

if child_age >= majority and child_not_capable:
  give lar-consent

if child_age >= majority and child_is_capable:
  give lar-consent-assent
```